### PR TITLE
Fix for Logstash 5.0.0 compatibility issue

### PR DESCRIPTION
**Pull request by phill84**
fix for #8

### CODE REVIEW

File: logstash-filter-alter.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-filter-alter'
-  s.version         = '2.0.5'
+  s.version         = '2.0.6'
```
Comment (wiibaa) [inline logstash-filter-alter.gemspec]: It is preferred to do a major bump, can you please set it to 3.0.0

File: logstash-filter-alter.gemspec [hunk 2]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "filter" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core-plugin-api", "~> 1.0"
+  s.add_runtime_dependency "logstash-core-plugin-api", "~> 2.0"
```
Comment (wiibaa) [inline logstash-filter-alter.gemspec]: The current pattern is to use `s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"` to keep support for LS 2.4.x && 5.x

File: spec/filters/alter_spec.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 
-require "logstash/devutils/rspec/spec_helper"
+require_relative '../spec_helper'
```
Comment (wiibaa) [inline spec/filters/alter_spec.rb]: just curious, why move this require into the new file spec_helper ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+# 2.0.6
```
Comment (wiibaa) [inline CHANGELOG.md]: As noted in the gemspec, should be 3.0.0

File: spec/filters/alter_spec.rb (hunk 3 above)
Comment (phill84) [inline spec/filters/alter_spec.rb]: I made the update based on what is generated by logstash-plugin from logstash 5.0.0, didn't notice this change.
Should it be reverted?

Comment (wiibaa) [inline spec/filters/alter_spec.rb]: as said, just curious, most of the plugins in logstash-plugins were done prior to the plugin generator. maybe @ph have a stronger point of view here ?

**Pull request merged**